os/include/tinyara/os_api_test_drv.h

```diff
@@ -30,18 +30,18 @@
 #include <tinyara/config.h>
 #include <tinyara/fs/ioctl.h>
 
-#ifdef CONFIG_DRIVERS_KERNEL_TEST
+#ifdef CONFIG_DRIVERS_OS_API_TEST
 
 /****************************************************************************
  * Pre-processor Definitions
  ****************************************************************************/
 /* Configuration
  *
- * CONFIG_DRIVERS_KERNEL_TEST - Enables kernel_test driver support
+ * CONFIG_DRIVERS_OS_API_TEST - Enables os_api_test driver support
  */
 
 /* IOCTL Commands */
-/* The kernel_test_drv module uses ioctl commands to identify the kernel module for which
+/* The os_api_test_drv module uses ioctl commands to identify the kernel module for which
```

Comment:
Please change the description also, because, as you said, this is used not only for kernel module but also for network and fs.